.env-mainnet-sample

```diff
@@ -67,3 +67,6 @@ EXTERNAL_LTC_ADDRESS=sample.ip.address
 #
 COMPOSE_PATH_SEPARATOR=:
 COMPOSE_FILE=docker-compose.yml:docker-compose.bitcoind.yml:docker-compose.litecoind.yml:docker-compose.override.yml
+
+# Path to rpc certs and identity keys
+SECURE_PATH=~/.sparkswap/secure
```

Comment:
[nit] suchhh a nit, but would rather all other information above the `COMPOSE_FILE` lines (having the COMPOSE lines be the last lines of the file)

Reply:
agree